alerts/pg.libsonnet

```diff
@@ -0,0 +1,27 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'pg-alert.rules',
+        rules: [
+          {
+            alert: 'OCS_CephPgStuck',
+            expr: |||
+              max(ceph_osd_numpg{%(cephExporterSelector)s}) - scalar(ceph_pg_active{%(cephExporterSelector)s}) <= 0
```

Comment:
Shouldn't this rule be like `max(ceph_osd_numpg{job="rook-ceph-mgr"}) - scalar(ceph_pg_active{job="rook-ceph-mgr"}) >= 0` or we need to take rate of this?

@pcuzner comments?